src/components/VmActions/actions.js

```diff
@@ -0,0 +1,20 @@
+import { SET_CONSOLE_IN_USE, CONSOLE_IN_USE } from './constants'
```

Comment:
I would prefer a single file to be used across whole application. 
It will help to avoid name conflicts without explicit namespacing.
Merge conflicts are easy to resolve.